vnc/vnc.h

```diff
@@ -133,8 +133,8 @@ struct vnc
     int mod_mouse_state;
     int palette[256];
     int vnc_desktop;
-    char username[256];
-    char password[256];
+    char username[8192];
```

Comment:
Does this come from VNC protocol specification?

Reply:
RealVNC (authoritative) is currently 255 characters. Older legacy implementations may still be 8. RFB protocol itself is agnostic[0]. Since the user password is used as the DES key, and DES key size is limited to 56 bits (plus 8 parity bits), your key can only be up to 7 8-byte characters long. If the password is shorter than 8 characters, it's just padded with zeroes.

But let me rebase, as you are looking at an older commit